module/spl/spl-condvar.c

```diff
@@ -312,10 +312,27 @@ cv_timedwait_hires(kcondvar_t *cvp, kmutex_t *mp, hrtime_t tim, hrtime_t res,
 	if (!(flag & CALLOUT_FLAG_ABSOLUTE))
 		tim += gethrtime();
 
-	return (__cv_timedwait_hires(cvp, mp, tim, TASK_UNINTERRUPTIBLE));
+	return (__cv_timedwait_hires(cvp, mp, tim, state));
+}
+
+hrtime_t
+cv_timedwait_hires(kcondvar_t *cvp, kmutex_t *mp, hrtime_t tim, hrtime_t res,
+    int flag)
+{
+	return (cv_timedwait_hires_common(cvp, mp, tim, res, flag,
+	    TASK_UNINTERRUPTIBLE));
 }
 EXPORT_SYMBOL(cv_timedwait_hires);
 
+hrtime_t
+cv_timedwait_hires_sig(kcondvar_t *cvp, kmutex_t *mp, hrtime_t tim, hrtime_t res,
```

Comment:
We should call this `cv_timedwait_sig_hires()` which is consistent with what it's called in [illumos](https://github.com/openzfs/openzfs/blob/master/usr/src/uts/common/os/condvar.c#L363).